packages/@storefront/core/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
+### Changed
+- SF-1315: Allow users to set `query` to empty string within the application store.
```

Comment:
You should mention something about this being in the URL service, specifically the parser.

Reply:
Will do.